modules/installation-vsphere-machines.adoc

```diff
@@ -6,28 +6,23 @@
 [id="installation-vsphere-machines_{context}"]
 = Creating {op-system-first} machines in vSphere
 
-Before you install a cluster that contains user-provisioned infrastructure on
-VMware vSphere, you must create {op-system} machines on vSphere hosts for it to
-use.
+Before you install a cluster that contains user-provisioned infrastructure on VMware vSphere, you must create {op-system} machines on vSphere hosts for it to use.
 
 .Prerequisites
 
 * Obtain the Ignition config files for your cluster.
-* Have access to an HTTP server that you can access from your computer and that
-the machines that you create can access.
+* Have access to an HTTP server that you can access from your computer and that the machines that you create can access.
 * Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
 
 .Procedure
 
-. Upload the bootstrap Ignition config file, which is named
-`<installation_directory>/bootstrap.ign`, that the installation program created
-to your HTTP server. Note the URL of this file.
+. Upload the bootstrap Ignition config file that the installation program created to your HTTP server.
 +
-You must host the bootstrap Ignition config file because it is too large to
-fit in a vApp property.
+* The file is named `<installation_directory>/bootstrap.ign`.
+* Note the URL of this file. You will need it for the following steps.
+* You must host the bootstrap Ignition config file because it is too large to fit in a vApp property.
 
-. Save the following secondary Ignition config file for your bootstrap node to
-your computer as `<installation_directory>/append-bootstrap.ign`.
+. Create a secondary Ignition config file and save it to your computer as `<installation_directory>/append-bootstrap.ign` using the following example:
```

Comment:
Is it no longer necessary to host `boostrap.ign` separately anymore? If not, then I think we can remove these first 2 steps. Is that correct?

Reply:
@bobfuru nope. I tested it last year [0].
IPI doesn't use the webserver and it hasn't caused an issue. I am fairly confident that this change change.

[0] - https://gist.github.com/jcpowermac/501a7db7e875acd2efb83bbb9250a02f

@patrickdillon do you agree?